support-data-rest/src/main/java/uk/ac/ebi/uniprot/api/disease/DiseaseDocumentToDiseaseConverter.java

```diff
@@ -0,0 +1,28 @@
+package uk.ac.ebi.uniprot.api.disease;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.ac.ebi.uniprot.cv.disease.Disease;
+import uk.ac.ebi.uniprot.json.parser.disease.DiseaseJsonConfig;
+import uk.ac.ebi.uniprot.search.document.disease.DiseaseDocument;
+
+import java.io.IOException;
+import java.util.function.Function;
+
+@Slf4j
+@Service
+public class DiseaseDocumentToDiseaseConverter implements Function<DiseaseDocument, Disease> {
```

Comment:
corresponding test for this class?

Reply:
Edd,
I have added many tests in DiseaseTest to test the ObjectMapper. That was the reason i had not written the test cases for this class. But i have added a test class with one test case the DiseaseDocumetnToDiseaseConverter class.